### PR TITLE
Feature add release plugin

### PR DESCRIPTION
**Pull request by petterarvidsson**

### CODE REVIEW

**Comment by petterarvidsson** [comment]
@nsg Ready for review!

File: bintray [hunk 1]
```diff
@@ -15,6 +15,16 @@ fi
 
 # Upload tagged releases to bintray
 if [ $TRAVIS_TAG ]; then
-	curl -T $ARTIFACT $API_AUTH $API_URL/content/konstructs/jars/server/$TRAVIS_TAG/konstructs-server-build-$TRAVIS_TAG.jar
+	# Release downloadable JAR
+        curl -T $ARTIFACT $API_AUTH $API_URL/content/konstructs/jars/server/$TRAVIS_TAG/konstructs-server-build-$TRAVIS_TAG.jar
```
Comment (nsg) [inline bintray]: The file was using tabs, you are using spaces ... :)

**Comment by nsg** [comment]
Looks good, except the indentation :)

File: bintray (hunk 1 above)
Comment (petterarvidsson) [inline bintray]: Since I am mostly programming my emacs will always use spaces. I'll change it :-)

**Comment by petterarvidsson** [comment]
@nsg Fixed the indentation (sorry)

**Comment by nsg** [comment]
@petterarvidsson :+1:

**Pull request merged**